sphinx/builders/htmlhelp.py

```diff
@@ -70,8 +70,8 @@
 
 project_template = '''\
 [OPTIONS]
-Binary TOC=No
-Binary Index=No
+Binary TOC=%(binary_toc)s
+Binary Index=%(binary_index)s
```

Comment:
What "Binary" means? Is there any description for these options?

Reply:
I write a long explanation, please read https://github.com/sphinx-doc/sphinx/issues/5788